pkg/storage/chunk/client/aws/s3_storage_client.go

```diff
@@ -39,11 +39,25 @@ import (
 const (
 	SignatureVersionV4 = "v4"
 	SignatureVersionV2 = "v2"
+
+	// S3 Storage Class options which define the data access, resiliency & cost requirements of objects
+	// https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutObject.html#API_PutObject_RequestSyntax
+	StorageClassGlacier                  = "GLACIER"
+	StorageClassDeepArchive              = "DEEP_ARCHIVE"
+	StorageClassGlacierInstantRetrieval  = "GLACIER_IR"
+	StorageClassIntelligentTiering       = "INTELLIGENT_TIERING"
+	StorageClassOneZoneInfrequentAccess  = "ONEZONE_IA"
+	StorageClassOutposts                 = "OUTPOSTS"
+	StorageClassReducedRedundancy        = "REDUCED_REDUNDANCY"
+	StorageClassStandard                 = "STANDARD"
+	StorageClassStandardInfrequentAccess = "STANDARD_IA"
```

Comment:
could you please remove code duplicates?  you added exactly the same variables to pkg/storage/bucket/s3/config.go . maybe create a new package with all common logic to a new package and re-use this new package in both packages... 
Does it make sense?

Reply:
Thanks Vlad, yes I agree it should be refactored - it did feel a bit dirty. Is there a package location you'd suggest?

Reply:
maybe create a package in `pkg/storage/common/aws`

Reply:
Duplicates have been refactored :)

Reply:
thanks 🚀